webview_panels/src/modules/queryPanel/useQueryPanelListeners.ts

```diff
@@ -15,13 +15,17 @@ import {
 } from "./context/queryPanelSlice";
 import useQueryPanelState from "./useQueryPanelState";
 import { panelLogger } from "@modules/logger";
-import { executeRequestInAsync } from "@modules/app/requestExecutor";
+import {
+  executeRequestInAsync,
+  executeRequestInSync,
+} from "@modules/app/requestExecutor";
 import { HINTS, HINT_VISIBILITY_DELAY } from "./constants";
 import { QueryPanelStateProps } from "./context/types";
 
-const useQueryPanelListeners = (): { loading: boolean } => {
+const useQueryPanelListeners = (): { loading: boolean; isPanel: boolean } => {
   const dispatch = useQueryPanelDispatch();
   const { loading, lastHintTimestamp, hintIndex } = useQueryPanelState();
+  const [isPanel, setIsPanel] = useState(true);
```

Comment:
this state var name and useQueryPanelListeners argument are conflicting. Do you know in run time, which variable will be used? and if you want useQueryPanelListeners argument to override the state variable, update the state variable using useEffect